source/simulator/core.cc

```diff
@@ -1693,6 +1693,16 @@ namespace aspect
               solve_advection(AdvectionField::composition(c));
             }
 
+          if (
+            parameters.use_discontinuous_composition_discretization
+            &&
+            parameters.use_limiter_for_discontinuous_solution
+          )
+            {
+              pcout << "Start to apply the limiter to the discontinuous solution " << std::endl;
```

Comment:
To keep the style of the rest of the screen output, I would suggest to change this to: "Applying bound preserving limiter to advection field ..." or something similar. Also please take a look at how many spaces the other lines prepend to their text to make the output look nicely aligned.